umad-02-keys-and-authentication.md

```diff
@@ -25,42 +25,56 @@ section.
 
 ## Public Key Exchange
 
-VASPs expose their public keys to other VASPs by responding to `GET` requests at the endpoint
+TThe UMA protocol relies on X.509 certificates for public key exchange among VASPs. In order to obtain an X.509
+certificate, the VASP is required to submit a request to a trusted certificate authority (CA), which issues the
+certificate. The issuing CA must keep track of certificates that are revoked, and provide this information to other
```

Comment:
I'm not 100% sure how CRLs work if there's no CA that signed the cert, but we should check with people who know more than I do.

Reply:
Yeah a CRL for a self-signed certificate doesn't make sense. To do this the VASP would have to generate their own CA certificate and use that to sign the actual certificate.